webcompat/static/js/lib/bugform.js

```diff
@@ -256,7 +256,13 @@ function BugForm() {
                   for (var item in issues.items) {
                     var issue = issues.items[item];
                     $("#bugList").append(
-                      `<li><a href='https://webcompat.com/issues/${issue.number}' target='_blank'>#${issue.number} - ${issue.title}</a></li>`
+                      "<li><a href='https://webcompat.com/issues/" +
```

Comment:
We have lodash loaded, it might be a bit nicer to read if we use `_.template`?

https://lodash.com/docs/3.10.1#template

something like:

```
(outside of for loop)
var makeLi = _.template("<li><a href='https://webcompat.com/issues/<%- issue.number %>' target='_blank'>#<%- issues.number %> - <%- issue.title %></a></li>")

$("#bugList").append(makeLi({'issue': issue})
```

Reply:
Thats a good idea, but CSP is blocking lodash here (which is weird).

Reply:
Ah right, we don't allow eval since we move to jst stuff. So that's another option.